src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-size: 12px;
+  box-sizing: border-box;
+  font-style: normal;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
```

```suggestion
  font-size: 12px;
  box-sizing: border-box;
  font-style: normal;
  font-family: "Roboto", sans-serif;
  font-weight: 500;
```
Всі ці властивості (крім ресету відступів) варто писати в тезі html